ApplETS/Notre-Dame
# Bugfix/fix duplicate course in dashboard

### ⁉️ Related Issue
closes: #510 

## 📖 Description
- Create an index for the displayed view. If displayed, the bottom bar will not update the view.
- Make the dashboard view refreshable

### 🧪 How Has This Been Tested?

1. Click multiple times on the same icon on the bottom bar the see that the page is not updating
2. Click on all bottom bar icons to see that they change
3. Refresh the dashboard view by swipping down the page

### ☑️ Checklist before requesting a review
- [x] Make sure golden files changes were reviewed and approved.

### 🖼️ Screenshots (if useful):
